src/main/java/Duke.java

```diff
@@ -1,10 +1,154 @@
+import java.util.Scanner;
+
+
 public class Duke {
+    public static int listCount=0;
+    static  Task[] list=new Task[100];
+    static final String BORDER="\n________________________________";
+    static final String BORDER_WITHOUT_SKIP="________________________________";
     public static void main(String[] args) {
+        /* To check if keywords like bye, list or done is used */
+        boolean byeDetected=false;
+        boolean commandsDetected=false;
+        Scanner in = new Scanner(System.in);
+        String input;
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+        String welcomeMessage = logo +
+                "________________________________\n"
+                +"Whats up people! Duke is here to save the day\n"
+                +"Soooo, what can I do for you?\n"
+                +"________________________________\n";
+
+        String byeMessage =
+                "Aye captain. This is Duke Signing out!\n"
+                +"________________________________";
```

Comment:
just a suggestion, maybe you could have a method to print your divider instead :D